cypress/support/cypress-commands/commands.js

```diff
@@ -658,9 +658,6 @@ Cypress.Commands.add('launchApp', (appType, appCallSign) => {
       ? UTILS.getEnvVariable(CONSTANTS.THIRD_PARTY_APP_ID)
       : UTILS.checkForSecondaryAppId(appCallSign); // this is for the app to know the appId used for launch, so that it can use the same for creating PubSub connection.
   // if appType is certification, the appLaunch is for certification purposes. In such a case, discovery.launch should go with a basic intent that has the appId and the certification app role.
-  // create the request map
-  // basic intent to be sent to the app on launch
```

Comment:
Add comment here 
// Creating data for basic intent to be sent to the app on launch

Reply:
Added comment